### PR TITLE
chore: test on both eslint 5 and 6

### PR DESCRIPTION
**Pull request by SimenB**

### CODE REVIEW

File: src/rules/__tests__/consistent-test-it.test.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import { TSESLint } from '@typescript-eslint/experimental-utils';
+import resolveFrom from 'resolve-from';
 import rule from '../consistent-test-it';
 import { TestCaseName } from '../utils';
 
 const ruleTester = new TSESLint.RuleTester({
-  parser: 'espree',
+  parser: resolveFrom(require.resolve('eslint'), 'espree'),
```
Comment (SimenB) [inline src/rules/__tests__/consistent-test-it.test.ts]: this allows us to not depend on `espree`. eslint 5 & 6 uses different versions, so it'd be painful

Comment (G-Rath) [reply src/rules/__tests__/consistent-test-it.test.ts]: What's the background for supporting 5 & 6? (Just for my personal info)

Comment (SimenB) [reply src/rules/__tests__/consistent-test-it.test.ts]: no reason not to support 5

Comment (G-Rath) [reply src/rules/__tests__/consistent-test-it.test.ts]: Oh, the pain you're referring to is w/ if we have a dependency on a specific version 5 or 6, for consumers of the library.

Sweet as :)

Comment (SimenB) [reply src/rules/__tests__/consistent-test-it.test.ts]: it'd be painful for us to install both v4 and v5 of espree depending on which version of eslint we were testing against

**Pull request merged**